.vib/argo-cd/vib-publish.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/argo-cd"
         },
-        "runtime_parameters": "ImNvbmZpZyI6CiAgInNlY3JldCI6CiAgICAiYXJnb2NkU2VydmVyQWRtaW5QYXNzd29yZCI6ICJLak1mZHpjamtwM0siCiJyZWRpcyI6CiAgImF1dGgiOgogICAgInBhc3N3b3JkIjogImpuZGZERmYiCiJzZXJ2ZXIiOgogICJzZXJ2aWNlIjoKICAgICJwb3J0cyI6CiAgICAgICJodHRwIjogODAKICAgICJ0eXBlIjogIkxvYWRCYWxhbmNlciI=",
+        "runtime_parameters": "Y29uZmlnOgogIGtub3duSG9zdHM6IHwKICAgIGdpdGh1Yi5jb20gc3NoLXJzYSBBQUFBQjNOemFDMXljMkVBQUFBQkl3QUFBUUVBcTJBN2hSR21kbm05dFVEYk85SURTd0JLNlRiUWErUFhZUENQeTZyYlRyVHR3N1BIa2NjS3JwcDB5VmhwNUhkRUljS3I2cExsVkRCZk9MWDlRVXN5Q09WMHd6ZmpJSk5sR0VZc2RsTEppekhoYm4ybVVqdlNBSFFxWkVUWVA4MWVGekxRTm5QSHQ0RVZWVWg3VmZERVNVODRLZXptRDVRbFdwWExtdlUzMS95TWYrU2U4eGhIVHZLU0NaSUZJbVd3b0c2bWJVb1dmOW56cElvYVNqQit3ZXFxVVVtcGFhYXNYVmFsNzJKK1VYMkIrMlJQVzNSY1QwZU96UWdxbEpMM1JLclRKdmRzakUzSkVBdkdxM2xHSFNaWHkyOEczc2t1YTJTbVZpL3c0eUNFNmdiT0RxblRXbGc3K3dDNjA0eWRHWEE4VkppUzVhcDQzSlhpVUZGQWFRPT0KICBzZWNyZXQ6CiAgICBjcmVhdGU6IHRydWUKICAgIGFyZ29jZFNlcnZlckFkbWluUGFzc3dvcmQ6IENvbXBsaWNhdGVkUGFzc3dvcmQxMjMhNApjb250cm9sbGVyOgogIHNlcnZpY2U6CiAgICBwb3J0OiA4MDgzCnNlcnZlcjoKICBwb2RTZWN1cml0eUNvbnRleHQ6CiAgICBlbmFibGVkOiB0cnVlCiAgICBmc0dyb3VwOiAxMDAyCiAgY29udGFpbmVyU2VjdXJpdHlDb250ZXh0OgogICAgZW5hYmxlZDogdHJ1ZQogICAgcnVuQXNVc2VyOiAxMDAyCiAgICBhbGxvd1ByaXZpbGVnZUVzY2FsYXRpb246IGZhbHNlCiAgICBjYXBhYmlsaXRpZXM6CiAgICAgIGRyb3A6CiAgICAgIC0gYWxsCiAgaW5zZWN1cmU6IGZhbHNlCiAgY29uZmlnRW5hYmxlZDogdHJ1ZQogIGNvbnRhaW5lclBvcnRzOgogICAgaHR0cDogODA4MAogICAgaHR0cHM6IDg0NDMKICBzZXJ2aWNlOgogICAgcG9ydHM6CiAgICAgIGh0dHA6IDgxCiAgICAgIGh0dHBzOiA0NDMKICAgIHR5cGU6IExvYWRCYWxhbmNlcgogIHNlcnZpY2VBY2NvdW50OgogICAgY3JlYXRlOiB0cnVlCiAgICBhdXRvbW91bnRTZXJ2aWNlQWNjb3VudFRva2VuOiB0cnVlCnJlcG9TZXJ2ZXI6CiAgc2VydmljZToKICAgIHBvcnQ6IDgwODAKcmVkaXM6CiAgZW5hYmxlZDogdHJ1ZQogIGF1dGg6CiAgICBlbmFibGVkOiB0cnVlCiAgICBwYXNzd29yZDogc29tZUNvbXBsaWNhdGVkUGFzczEyMzQ1IQogIGFyY2hpdGVjdHVyZTogc3RhbmRhbG9uZQ==",
```

Comment:
I think you forgot to add:

-  `controller.clusterAdminAccess=true`
- `rbac.create=true`